core/master/src/main/resources/config/commands/ftp/core.conf

```diff
@@ -410,21 +410,60 @@ help Add a new user
 help.specific ${command} <user> <password> [<ident@ip#1> ...]
 }
 
+SITE ADDGROUP {
+plugin org.drftpd.commands.usermanagement
+class GroupManagementHandler
+method doSITE_ADDGROUP
+perms =siteop
+help Add a new group
+help.specific ${command} <group>
+}
+
+# Backwards compatibility
 SITE CHANGE {
 plugin org.drftpd.commands.usermanagement
 class UserManagementHandler
-method doSITE_CHANGE
+method doSITE_CHANGEUSER
 perms =gadmin =siteop
 help Change parameters for a user
 help.specific ${command} <username> <field> <value>
 help.specific Valid fields: group, created, ratio, idle_time, credits, tagline
-help.specific num_logins # [opt perIP], comment, group_ratio #min #max
-help.specific group_slots # [opt leech slots], wkly_allotment, max_sim #dn #up
+help.specific num_logins # [opt perIP], comment, wkly_allotment, max_sim #dn #up
 }
 
-SITE CHGRP {
+SITE CHANGEUSER {
 plugin org.drftpd.commands.usermanagement
 class UserManagementHandler
+method doSITE_CHANGEUSER
+perms =gadmin =siteop
+help Change parameters for a user
+help.specific ${command} <username> <field> <value>
+help.specific Valid fields: group, created, ratio, idle_time, credits, tagline
+help.specific num_logins # [opt perIP], comment, wkly_allotment, max_sim #dn #up
+}
+
+SITE CHANGEGROUP {
+plugin org.drftpd.commands.usermanagement
+class GroupManagementHandler
+method doSITE_CHANGEGROUP
+perms =siteop
+help Change parameters for a group
+help.specific ${command} <groupname> <field> <value>
+help.specific Valid fields: created, slots, leechslots
+}
+
+SITE CHANGEGROUPADMIN {
+plugin org.drftpd.commands.usermanagement
+class GroupManagementHandler
+method doSITE_CHANGEGROUPADMIN
+perms =siteop
+help Add/Remove users as group admin from a group
+help.specific ${command} <group> <user> [<user> ...]
+}
+
+SITE CHGRP {
+plugin org.drftpd.commands.usermanagement
+class GroupManagementHandler
 method doSITE_CHGRP
```

Comment:
Difference with doSITE_CHANGEGROUP?

Reply:
Yes
CHANGEGROUP changes parameters on a group.
CHGRP adds/removes a user to/from a group